drivers/wifi/nrf700x/zephyr/inc/zephyr_coex_struct.h

```diff
@@ -16,66 +16,108 @@
 
 /* Max size of message buffer (exchanged between host and MAC). This is in "bytes" */
 #define MAX_MESSAGE_BUF_SIZE 320
-/* Number of elements in coex_ch_configuration other than configbuf[] */
+/* Number of elements in nrf_wifi_coex_ch_configuration other than configbuf[] */
 #define NUM_ELEMENTS_EXCL_CONFIGBUF 4
 /* Each configuration value is of type uint32_t */
 #define MAX_NUM_CONFIG_VALUES ((MAX_MESSAGE_BUF_SIZE-\
 	(NUM_ELEMENTS_EXCL_CONFIGBUF*sizeof(uint32_t)))>>2)
-/* Number of elements in coex_sr_traffic_info other than sr_traffic_info[] */
+/* Number of elements in nrf_wifi_coex_sr_traffic_info other than sr_traffic_info[] */
 #define NUM_ELEMENTS_EXCL_SRINFOBUF 1
 /* Each SR Traffic Info is of type uint32_t */
 #define MAX_SR_TRAFFIC_BUF_SIZE 32
 
+/* Indicates the type of priority window i.e., WLAN or SR */
+enum {
+	NRF_WIFI_WIFI_WINDOW = 0,
```

Comment:
IMHO, if you are not expecting more things to be added here, you can just convert them to boolean and call `is_X`. And even if you think more will be added, these shouldn't be anonymous enums and should aptly describe the grouping.

Same for others.